test/coverage/communication/SubscriberTest.cpp

```diff
@@ -122,7 +122,7 @@ TEST_F(SubscriberTest, SubscribeSuccess) {
 	EXPECT_TRUE(MsgDiff::Equals(testTopicUUri_, transport->sink_filter_));
 
 	const size_t max_count = 100;
```

Comment:
Minor: use constexpr
```suggestion
	constexpr size_t max_count = 100;
```